src/rust/ide/view/src/documentation.rs

```diff
@@ -56,6 +63,9 @@ enum InputFormat {
 
 pub use visualization::container::overlay;
 
+#[allow(unused_qualifications)]
```

Comment:
why?

Reply:
tbh this is only to surpass linter inadeequate warning as the type signatuure there is really necessary ...

Reply:
Almost impossible. Please investigate that before merge and write to me what you found out with details. This should not be used. If this is really needed, create bug report on Rust linter repo and link it here in a FIXME comment